plugins/generic/backup/locale/ar_IQ/locale.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/backup/locale.xml
```

Comment:
No worries, I was just pointing out that the path indicated in the comment is wrong (was wrong in the English version, too). No problem.